pkgs/applications/editors/kakoune/plugins/update.py

```diff
@@ -50,7 +50,7 @@
 lib.filterAttrs (n: v: v != null) checksums
 )"""
 
-HEADER = "# This file has been generated by ./pkgs/applications/editors/kakoune/plugins/update.py. Do not edit!"
+HEADER = "# This file has been @generated by ./pkgs/applications/editors/kakoune/plugins/update.py. Do not edit!"
```

Comment:
why this `@` ?

Reply:
Just saw that the same does poetry

https://github.com/python-poetry/poetry/pull/2773

Reply:
Ok, fine by me.